src/components/shared/formItems/ProjectHandle.tsx

```diff
@@ -80,16 +89,31 @@ export default function ProjectHandle({
     [handleExists, requireState],
   )
 
-  let suffix: string | JSX.Element = ''
-  if (handleExists && requireState === 'notExist') {
-    suffix = 'Handle already in use'
-  }
-  if (handleExists === false && requireState === 'exists') {
-    suffix = inputContents ? 'Handle not found' : ''
-  }
-  if (handleExists && requireState === 'exists') {
-    suffix = <CheckCircleOutlined style={{ color: colors.icon.success }} />
-  }
+  const suffix: string | JSX.Element = useMemo(() => {
```

Comment:
Changed `suffix` to memoized value. I _think_ this is a good use case, please correct me if not.

Reply:
my understanding is that memoization is kinda subjective—just meant to be used to prevent expensive logic (i.e. network calls or complex calculations) from being run unnecessarily. i'm down to use it here